db/migrations/tbd_add_validator_stats_total_accumulation_column copy.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query with new column total_accumulation_exported for validator_stats_status';
+ALTER TABLE validator_stats_status ADD COLUMN IF NOT EXISTS total_accumulation_exported BOOLEAN NOT NULL DEFAULT FALSE;
+
+SELECT 'setting new validator_stats_status columns to true for already exported days'; 
+UPDATE validator_stats_status
+SET 
+    total_accumulation_exported = true
+WHERE total_performance_exported=true;
```

Comment:
Please use `value = const` or `value=const` but not both, this hurts my eyes.

Reply:
added some spaces, don't want your eyes to get hurt ;-)